src/css/custom.css

```diff
@@ -76,3 +76,111 @@ img + br + em::before {
     flex: 1;
     min-width: 20rem;
 }
+
+.typography h1 {
+    font-family: "Segoe UI", "Helvetica Neue", "Droid Sans", Helvetica, Arial,
+        sans-serif;
+    margin: 0px 0px 0.613rem !important;
```

Comment:
All of these !important tags hurts my brain. Are they needed or can we increase the specificity to not need them?

Reply:
This CSS is temporary.

Reply:
To elaborate I'll be moving styling over to the system we use for buttons etc...